dev/Repeater/InteractionTests/RepeaterTests.cs

```diff
@@ -22,6 +27,64 @@ namespace Windows.UI.Xaml.Tests.MUXControls.InteractionTests
     [TestClass]
     public class RepeaterTests
```

Comment:
Thank you for fixing this bug. I do not think you need an interaction test for this. It would be a lot simpler to just add an api test.

Reply:
Oh I see. Just for confirmation, when writing Api tests, I will need to create the ItemRepeater inside the test and can't use the test page I modified in the MUXControlsTestApp, right?

Reply:
That is correct. You can keep the app changes you have made if you like, but yes, the api tests do not use the test app pages.

Reply:
I think we may need an interaction test. I have tried finding a way to replicate the steps to reproduce the bug in an api test, however since there seems to be no way to invoke the button click, I had to choose a workaround. But removing the bug fix did not result in the newly added api test to fail. I pushed the test, maybe you can find what is going wrong. 😅

Otherwise it might be easier to stick with the interaction test, as this seems to work correctly.

Reply:
The Api test should work now, so we do not need the interaction test anymore, but since you added the first test and the scaffolding which will be useful, I will leave it to you to decide if you want to keep it or remove it.

Reply:
I think we should remove the interaction test, but leave the rest of the test class the way it is, just in case we need an interaction test for the repeater.